paperqa/types.py

```diff
@@ -456,8 +456,11 @@ def inject_clean_doi_url_into_data(data: dict[str, Any]) -> dict[str, Any]:
     def remove_invalid_authors(cls, data: dict[str, Any]) -> dict[str, Any]:
         """Capture and cull strange author names."""
         if authors := data.get("authors"):
+            # On 10/29/2024 while indexing 19k PDFs, a provider (unclear which one)
```

Comment:
Is there a link to the failed run that you can put in the comment.

Reply:
I do have a shareable link, but it's to our internal Weights & Biases account, so it doesn't make sense to add to the code.

What I did is put the failure's trace in this PR's description, let me know if you think I can do something better